tjreports/site/controllers/reports.php

```diff
@@ -350,4 +352,45 @@ public function getreport()
 			jexit();
 		}
 	}
+
+/**
+	* Function used to delete reports
+	*
+	* @return  boolean
+	*
+	* @since  1.0
+	*/
+	public function deleteQuery()
+	{
+		$cid = JFactory::getApplication()->input->get('cid', '', 'array');
+		$model = JModelLegacy::getInstance('Report', 'TjreportsModel');
+
+		if ($model->delete($cid))
+		{
+			echo json_encode(1);
+			jexit();
```

Comment:
@hemantpurohit1  jexit() should be added in both true and false cases. in case of failure, echo json_encode(0)